tools/scene.py

```diff
@@ -125,23 +126,34 @@ def prettify(self, encoding=None, formatter='minimal', indent_width=4):
     if not os.path.exists(args.root):
         os.makedirs(args.root)
 
+    # Get the interactive tool path
+    interactive_dir = pathlib.Path(os.path.dirname(__file__)).parents
+    if len(interactive_dir) == 0:
+        interactive_dir = ".."
+    else:
+        interactive_dir = interactive_dir[0]
+
+    print(f'Tool directory: {interactive_dir}')
     main_index_fname = os.path.join(args.root, 'index.html')
+    print(f'Main HTML path: {main_index_fname}')
     if not os.path.exists(main_index_fname):
         # Copy the dumy main index
         shutil.copy(os.path.join(os.path.dirname(__file__), 'index.html'), main_index_fname)
 
+    # Copy utils is there missing
```

Comment:
This is an optional addition: if `<root_dir>/utils` directory does not exist, copy from the interactive-viewer.